packages/nextjs-demo/src/components/ResultContainer.tsx

```diff
@@ -17,7 +17,7 @@ export default function ResultContainer({
             <h3 className="text-base font-semibold leading-6 text-gray-900">{title}</h3>
             <p>{description}</p>
           </div>
-          <div className="bg-white px-4 py-5 sm:px-6">{children}</div>
+          <div className="bg-white px-4 py-5 sm:px-6 whitespace-pre-wrap">{children}</div>
```

Comment:
Ha! Super clever and simple way to make the demo look dramatically better.

Reply:
who needs HTML when you have "\n"?